pkg/kubectl/cmd/util/factory_builder.go

```diff
@@ -51,17 +50,9 @@ func NewBuilderFactory(clientAccessFactory ClientAccessFactory, objectMappingFac
 func (f *ring2Factory) PrinterForCommand(cmd *cobra.Command, isLocal bool, outputOpts *printers.OutputOptions, options printers.PrintOptions) (printers.ResourcePrinter, error) {
```

Comment:
> isLocal bool

It seems we don't use it any more. We can drop it.

Reply:
Will do that in a follow up - was going to do the following:

1. remove isLocal
2. Collapse OutputOptions into PrintOptions
3. show how that is used by server-side get rendering

Reply:
> 2. Collapse OutputOptions into PrintOptions

PR currently open for this: https://github.com/kubernetes/kubernetes/pull/54811